resource/ui/ts/models/timeseries.ts

```diff
@@ -43,68 +44,183 @@ module Models {
         /**
          * QueryResultSet matches the successful output of the /ts/query
          * endpoint. This needs to be kept in sync with the
-         * TimeSeriesQueryResult protobuffer message on the server.
+         * TimeSeriesQueryResponse protobuffer message on the server.
          * (/protos/timeseries.proto).
          */
         export interface QueryResultSet {
```

Comment:
I was looking at what was needed to create a basic level of typescript definitions based on protobuf definitions and it looks pretty simple.  Would probably be a good open source project to get out there at some point.  For now, can we just pack them in the same module?